frontend/document/credocument.lua

```diff
@@ -1242,10 +1242,10 @@ function CreDocument:setBackgroundImage(img_path) -- use nil to unset
     self._document:setBackgroundImage(img_path)
 end
 
-function CreDocument:findText(pattern, origin, reverse, caseInsensitive)
-    logger.dbg("CreDocument: find text", pattern, origin, reverse, caseInsensitive)
+function CreDocument:findText(pattern, origin, reverse, caseInsensitive, page, regex)
```

Comment:
(Thought you added this unused `page` by error - but no, it's in the signature of the ones used with PDF/Kopt.)